emcc

```diff
@@ -689,6 +689,13 @@ try:
     pre_js += open(shared.path_from_root('src', 'emrun_prejs.js')).read() + '\n'
     post_js += open(shared.path_from_root('src', 'emrun_postjs.js')).read() + '\n'
 
+  if tracing:
+    dlMallocPath = shared.path_from_root('system/lib', 'dlmalloc.c')
+    if not newargs.__contains__(dlMallocPath):
+      tempDLMalloc = in_temp('dlmalloc.bc')
+      subprocess.call([shared.EMCC, dlMallocPath, '-o', tempDLMalloc, '--tracing'])
```

Comment:
i think you want to call `execute(` here, see other uses in that file. it checks the return code.